### PR TITLE
DEV-3889-solution of weareuniaue

### PR DESCRIPTION
**Pull request by zainabdnaya**

### CODE REVIEW

File: solutions/weareunique.go [hunk 1]
```diff
@@ -0,0 +1,28 @@
+package solutions
+
+import "strings"
+
+func Weareunique(str1 string, str2 string) int {
+
```
Comment (Hamzaelkhatri) [inline solutions/weareunique.go:6]: remove this empty line

**Review by Hamzaelkhatri — CHANGES_REQUESTED** [review]

File: solutions/weareunique.go (hunk 1 above)
Comment (zainabdnaya) [reply solutions/weareunique.go]: done

File: tests/unzipstring_test/main.go [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+		"5w6	2t",
+		"8o2d7-1a",
+		"4	1w8*6x",
+		"2p6t5z",
+		"7w1mw0q",
+		"0o5d3j",
+		"3a2b1c",
+	}
+	for _, s := range arr {
+		challenge.Function("Unzipstring", student.Unzipstring, solutions.Unzipstring, s)
```
Comment (Hamzaelkhatri) [inline tests/unzipstring_test/main.go:31]: u need to add two params here , `str1,str2`

**Review by Hamzaelkhatri — CHANGES_REQUESTED** [review]

File: tests/unzipstring_test/main.go (hunk 2 above)
Comment (zainabdnaya) [reply tests/unzipstring_test/main.go:31]: done

**Review by Hamzaelkhatri — APPROVED** [review]
LGTM!

File: solutions/weareunique.go [hunk 3]
```diff
@@ -0,0 +1,20 @@
+package solutions
```
Comment (MSilva95) [inline solutions/weareunique.go:1]: its not returning -1 when it should.
How did this get here with a wrong solution? Was it not tested?

**Review by MSilva95 — CHANGES_REQUESTED** [review]

File: solutions/weareunique.go (hunk 3 above)
Comment (zainabdnaya) [reply solutions/weareunique.go:1]: The subject is wrong because I meant to write that I should return ` 0` if there are no unique characters.

Comment (MSilva95) [reply solutions/weareunique.go:1]: not fixed, now it returns 0 where it should return -1
Test it first before you send it to review, don't feel like you have to do this in a hurry...

**Review by MSilva95 — CHANGES_REQUESTED** [review]

File: solutions/weareunique.go (hunk 3 above)
Comment (zainabdnaya) [reply solutions/weareunique.go:1]: Yes, You are right I didn't push the change , My bad sorry

File: tests/weareunique_test/main.go [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+import (
+	student "student"
+
+	"github.com/01-edu/go-tests/lib/challenge"
+	"github.com/01-edu/go-tests/solutions"
+)
+
+func main() {
+	table := [][]string{
+		{"foo", "boo"},
```
Comment (MSilva95) [inline tests/weareunique_test/main.go:12]: You are repeating a lot of tests... Why?

**Review by MSilva95 — CHANGES_REQUESTED** [review]

File: tests/weareunique_test/main.go (hunk 4 above)
Comment (zainabdnaya) [reply tests/weareunique_test/main.go:12]: corrected

Comment (MSilva95) [reply tests/weareunique_test/main.go:12]: there is still repetition i the test...

Comment (zainabdnaya) [reply tests/weareunique_test/main.go]: fixed

Comment (MSilva95) [reply tests/weareunique_test/main.go]: is it?
I can still see one more repetition on the test...
The test case {"", ""} is there twice...
This is one of the points @davhojt covered in the presentation about the reviews. please be more careful with this as there is no need for me to review this 3 times just to check if all the repetition is gone or not.

Comment (zainabdnaya) [reply tests/weareunique_test/main.go]: all the repetition is gone Now

**Review by jotapero — APPROVED** [review]
Looks Good :alien:

**Review by tiagocollot — APPROVED** [review]

**Review by tiagocollot — APPROVED** [review]
LGTM ⚡

**Pull request merged**